recipes/pyspark-stubs/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "pyspark-stubs" %}
+{% set version = "2.4.0.post5" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "2249e20f3d120551d6ba3bf8bca67c1cb9a2d2e39bd9ebcd3cb00ae906428f21"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  skip: true  # [py<36]
```

I think this recipe can be ``noarch``, you will just need to select ``python`` on ``host`` and ``run``
```suggestion
  noarch: python
```

---

Thanks for the review @marcelotrevisani!

Makes sense. I assume that https://github.com/conda-forge/staged-recipes/pull/9480#discussion_r322171542 is sufficient replacement (some of the constructs are not compatible with 3.5 and before).